### ISSUE TITLE
Deprecation: createClass & failed proptypes

### ISSUE THREAD

**Issue by alechp** [nitin42/glamorous-primitives#5]
Output on clean install / render:

![screen shot 2017-10-03 at 3 11 16 pm](https://user-images.githubusercontent.com/3058939/31151655-4e9df486-a84d-11e7-8ae1-617c73c7c637.png)

**Comment by nitin42** [comment]
Yeah it gives error when rendering to sketch. I'll try to fix it later this week.

**Comment by nitin42** [comment]
Well, this package depends on `react-primitives` for binding the aliases to the `glamorous` constructor. Seems like `react-primitives` relies on `createClass` method and the author has not updated that. As soon as he updates the codebase (with React 16), I'll fix this issue 😄
